gpu_requirements.txt

```diff
@@ -0,0 +1,10 @@
+torch==1.9.0
+torchvision
+pytorch-ignite==0.3.0
```

Comment:
not needed, I guess it is only for training right ?

Reply:
right, I've removed it